pyproject.toml

```diff
@@ -114,15 +114,61 @@ test = [
 build = [
     "build>=1.2.2.post1",
     "twine>=6.1.0",
+    "versioningit>=3.3.0",
 ]
 
-[tool.setuptools.packages.find]
-where = ["."]
-include = ["disnake*"]
-
 [tool.uv]
 required-version = ">=0.9.2"
 
+[tool.hatch.build]
+include = [
+    "disnake/",
+]
```

Comment:
*perhaps* worth noting, there doesn't seem to be a single python file in the built wheel besides `_version.py`:
<img width="818" height="156" alt="image" src="https://github.com/user-attachments/assets/30866597-d717-48a2-a97c-219e2572f75f" />

Reply:
Seems like this is also a shortcoming in CI. This should *absolutely* be discovered on the built wheels.